lib/lib_phpQuery.php

```diff
@@ -145,7 +145,7 @@ class DOMDocumentWrapper {
 	public $id;
 	/**
 	 * @todo Rewrite as method and quess if null.
-	 * @var unknown_type
```

Comment:
I only did a minimum in that library but there are many other problems, and it should soon be fully replaced